src/test/scala/lantern/TestTensorDifferentiation.scala

```diff
@@ -194,7 +194,7 @@ class AdLMSVectorTest extends LanternFunSuite {
       @virtualize
       def snippet(a: Rep[String]): Rep[Unit] = {
         val idx = var_new(0)
-        val t = Tensor.fill(seq => { idx += 1; idx }, 2, 3)
+        val t = Tensor.fill(Seq(2, 3), (seq => { idx += 1; idx }))
 
```

Comment:
I'd prefer writing this as

    val t = Tensor.fill(2, 3) { is => idx += 1; idx }